dev/NavigationView/NavigationView.cpp

```diff
@@ -1465,9 +1465,15 @@ void NavigationView::UpdatePaneLayout()
         const auto totalAvailableHeight = [this]() {
             if (const auto &paneContentRow = m_itemsContainerRow.get())
             {
-                // 20px is the padding between the two item lists
-                if (const auto &paneFooter = m_leftNavFooterContentBorder.get())
+                // We have 9px margin at the bottom, that's why we subtract 9 everywhere
+                if (m_menuItemsSource && m_footerItemsSource && m_menuItemsSource.Count() * m_footerItemsSource.Count() == 0 && !IsSettingsVisible())
                 {
+                    // Either of these collections has no items, let it take all of the available space then.
+                    return paneContentRow.ActualHeight() - 9;
```

Comment:
this hurts to look at.. but it seems like we were already doing it..  (can we get this value from the Margin property instead of blindingly subtracting 9?)

Reply:
+1, that would be nicer

Reply:
Fixed now.